ci-operator/config/openshift/openshift-tests-private/tools/get_missed_upgrade_jobs.py

```diff
@@ -67,8 +67,11 @@ def get_missed_profiles(e2e_jobs, upgrade_jobs, mode='y'):
         'hypershift',
         'longduration',
         'mco',
+        'migrate',
+        'migration',
         'netobserv',
         'nokubeadmin',
+        'obo',
```

Comment:
aro/mco/obo, they are too short. We don't have profiles with words that has those abbr as part of it, but there is no guarantee we will not in the future.
Perhaps try to check with `-`, e.g, `re.search('-obo', job)` and `re.search('obo-', job)`

Reply:
Not a blocker. We can do that in this PR, or in a separate PR.

Reply:
we have:
`aws-ipi-ovn-ipsec-f2-obo`
I will add `-` before them and I will also add some other improvements, so hold on.
Thanks.